CMakeLists.txt

```diff
@@ -573,8 +576,10 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" AND NOT BUILD_WITH_CONDA)
     ENDIF(NOT PYTHONLIBS_FOUND)
 
 # -------------------------------- pcl ----------------------------------
```

Comment:
I think we should have this instead
`# ------------------- pcl (pointclouds) -------------------------`